src/Features/Core/Portable/InitializeParameter/AbstractAddParameterCheckCodeRefactoringProvider.cs

```diff
@@ -601,14 +603,14 @@ private TStatementSyntax CreateNullCheckStatement(DocumentOptionSet optionSet, S
                 nameofExpression);
         }
 
-        private static (string pre, string post) GetPreAndPostTextParts(string text)
+        private static (string? pre, string? post) GetPreAndPostTextParts(string text)
```

Comment:
💡 Based on use sites, `(string pre, string post)?` would be a less intrusive change here.